AccessibilitySnapshotCore.podspec

```diff
@@ -0,0 +1,21 @@
+Pod::Spec.new do |s|
```

Comment:
I had to create a new podspec to be able to import `AccessibilitySnapshotCore` into the code files with the previews.